External/Plugins/ASCompletion/Completion/ASGenerator.cs

```diff
@@ -41,6 +42,10 @@ public class ASGenerator
         static private MemberModel contextMember;
         static private bool firstVar;
 
+        static List<ICompletionListItem> known;
+
+        public static List<ICompletionListItem> KnownList => known;

```

Comment:
Well, there can be several reasons: not fully owning the PC being used, or not administrative rights to install it or some other locked environment, lack of available space to install it, not easy access to download it, not supported OS, etc.